Corpus/residential.py

```diff
@@ -364,11 +364,26 @@ def receptacles(self):
                 eq = Equipment(**set_appliances[app])
                 r_app = dict()
                 n_app = 0
-                # loop for all household mmembers
+                # loop for all household members
                 for i in counter:
-                    r_appi, n_appi = eq.simulate(nday, dow, self.clusters[i], self.occ[i])
-                    r_app = stats.sum_dict(r_app, r_appi)
-                    n_app += n_appi
+                    # if appliances are not shared the load of using the appliance can be summed for each householdmember
+                    if app in ('CordlessPhone','PC'): # it is assumed that only the cordlessphone and PC are not shared. Each person in the household (>12) has its own phone or PC.
+                        r_appi, n_appi = eq.simulate(nday, dow, self.clusters[i], self.occ[i])
+                        r_app = stats.sum_dict(r_app, r_appi)
+                        n_app += n_appi
```

Comment:
I agree that each occupant may nowadays own their own PC or cellphone . However this would mean that the household owns e.g. 2 PCs, and not just one, as is the case with the current implementation. Given that the activities related to the use of these appliances are from 2005, I think it is not straightforward to just assume 2-3 PCs instead of 1. A more general update to the model and inputs should be done, in my view. (Plus, I think that the CordlessPhone here is in fact the fixed home phone, not a cellphone)

Reply:
Make sure to take a realistic estimate of the power use for laptops. Normal laptops have about a 60 Wh batter. If you mildly load it, you can do about 2-3h with it before the battery is dead, meaning about 20 W on average. In practice the laptop is however not used most of the time so the power is probably < 1W.

Reply:
Indeed, the current appliance properties and way of modeling are not consistent with modern electronic appliances. Definitely something to improve, but better do it in a more systematic way, where all appliances are brought up to date.